ui/imports/shared/popups/SendModal.qml

```diff
@@ -125,171 +127,165 @@ StatusDialog {
-                    StatusAssetSelector {
-                        id: assetSelector
-                        anchors.verticalCenter: parent.verticalCenter
-                        anchors.right: parent.right
-                        assets: popup.selectedAccount.assets
-                        defaultToken: Style.png("tokens/DEFAULT-TOKEN@3x")
-                        getCurrencyBalanceString: function (currencyBalance) {
-                            return Utils.toLocaleString(currencyBalance.toFixed(2), popup.store.locale, {"currency": true}) + " " + popup.store.currentCurrency.toUpperCase()
+
+                Column {
+                    id: assetAndAmmountSelector
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.leftMargin: Style.current.xlPadding
+                    anchors.rightMargin: Style.current.xlPadding
+                    z: 1
+
+                    Row {
+                        spacing: 16
+                        StatusBaseText {
+                            anchors.verticalCenter: parent.verticalCenter
+                            text: qsTr("Send")
+                            font.pixelSize: 15
+                            color: Theme.palette.directColor1
                         }
-                        tokenAssetSourceFn: function (symbol) {
-                            return symbol ? Style.png("tokens/" + symbol) : defaultToken
+                        StatusListItemTag {
+                            height: 22
+                            width: childrenRect.width
+                            title: assetSelector.selectedAsset.totalBalance > 0 ? qsTr("Max: ") + (assetSelector.selectedAsset ? d.maxFiatBalance : "0.00") : qsTr("No balances active")
+                            closeButtonVisible: false
+                            titleText.font.pixelSize: 12
+                            color: d.errorMode ? Theme.palette.dangerColor2 : Theme.palette.primaryColor3
+                            titleText.color: d.errorMode ? Theme.palette.dangerColor1 : Theme.palette.primaryColor1
                         }
-                        searchTokenSymbolByAddress: function (address) {
-                            if(popup.selectedAccount) {
-                                return popup.selectedAccount.findTokenSymbolByAddress(address)
+                    }
+                    Item {
+                        width: parent.width
+                        height: amountToSendInput.height
+                        AmountInputWithCursor {
+                            id: amountToSendInput
+                            anchors.verticalCenter: parent.verticalCenter
+                            anchors.left: parent.left
+                            anchors.leftMargin: -Style.current.padding
+                            width: parent.width - assetSelector.width
+                            placeholderText: "0.00" + " " + assetSelector.selectedAsset.symbol
```

Comment:
Same here, decimal separator hardcoded, and also symbol hardcoded to be after the amount.